web/src/app/converters/proto-model-converter.ts

```diff
@@ -74,3 +76,42 @@ export function partialSurveyToProto(
     })
   );
 }
+
+/**
+ * Creates a proto rapresentation of a Job.
+ */
+export function jobToProto(job: Job): DocumentData {
+  return toDocumentData(
+    new Pb.Job({
+      id: job.id,
+      index: job.index,
+      name: job.name,
+      style: new Pb.Style({color: job.color}),
+      tasks: job.tasks
```

Comment:
There's a lot happening in this function, but I understand this is still draft - please refactor in final version?